README.md

```diff
@@ -1,2 +1,22 @@
-# aws-utilities
-various functions that can be used to help out with braket python SDK
+# quera-ahs-utils
+This python package is a collection of tools that can be used to program QuEra's nuetral **atom analog quantum simulator** (**ahs**). These tools are primarily targeted towards the usage of [Amazon's Braket quantum computing service](https://aws.amazon.com/braket/) . The Braket python software development kit can be found [here](https://github.com/aws/amazon-braket-sdk-python) along with some examples of how to use their service through a collection of examples from both [Braket](https://github.com/aws/amazon-braket-examples/tree/main/examples/analog_hamiltonian_simulation) and [QuEra](https://github.com/QuEraComputing/QuEra-braket-examples). 
+
+We would be remiss not to advertise our own Julia SDK for programming QuEra's **ahs**, [Bloqade](https://queracomputing.github.io/Bloqade.jl/dev/). Bloqade is a powerful [Julia](https://julialang.org/) package tool for programming and modeling neutral atom quantum computing. 
+
+## Installation
+The package can be installed via pip:
+
+    pip install quera-ahs-utils
+
+## Package contents
+
+**quera-ahs-utils** is broken up into 5 modules each dealing with specific tools summarized in the table below:
+|              module             |                                                       description                                                      |
+|:-------------------------------:|:----------------------------------------------------------------------------------------------------------------------:|
+| **quera_ahs_utils**.analysis    | A collection of functions to do basic analysis on shot results                                                         |
```

Comment:
```suggestion
| **quera_ahs_utils**.analysis    | Perform analysis on shot results                                                         |
```
Remove "A collection of functions" as well as "easily generate". Just need to say what the tools in the module do